ccd-definition/CaseEvent/CareSupervision/MultiState-nonprod.json

```diff
@@ -0,0 +1,19 @@
+[
```

Comment:
hmm, what about AC1, addFamilyManCaseNumber should be only in submitted state, non prod file should have addFamilyManCaseNumber with reduced precontintionstate